app/static/tests/e2e/fit.etest.ts

```diff
@@ -263,7 +263,7 @@ test.describe("Wodin App model fit tests", () => {
         await expect(await page.innerText("#fit-cancelled-msg")).toBe("Model fit was cancelled before converging");
     });
 
-    test.only("can mark sensitivity as out of date after fit", async ({ page }) => {
```

Comment:
I am unsure why this was added. It's been preventing running of other tests but `fit.etest.ts`.

Reply:
No idea, don't remember adding that, probably a mistake!